cmd/root.go

```diff
@@ -37,10 +37,12 @@ func newRootCmd(out io.Writer) *cobra.Command {
 	projectID := os.Getenv("GCP_PROJECT_ID")
 	emulatorHost := os.Getenv("PUBSUB_EMULATOR_HOST")
 	gcpCredentialFilePath := os.Getenv("GOOGLE_APPLICATION_CREDENTIALS")
+	ackDeadline := os.Getenv("PUBSUB_ACK_DEADLINE")
 	rootCmd.PersistentFlags().Bool("help", false, fmt.Sprintf("help for %s", rootCmd.Name()))
 	rootCmd.PersistentFlags().StringVarP(&projectID, projectFlagName, "p", projectID, "gcp project id (You can also set 'GCP_PROJECT_ID' to env variable)")
 	rootCmd.PersistentFlags().StringVarP(&emulatorHost, hostFlagName, "h", emulatorHost, "emulator host (You can also set 'PUBSUB_EMULATOR_HOST' to env variable)")
 	rootCmd.PersistentFlags().StringVarP(&gcpCredentialFilePath, credFileFlagName, "c", gcpCredentialFilePath, "gcp credential file path (You can also set 'GOOGLE_APPLICATION_CREDENTIALS' to env variable)")
+	rootCmd.PersistentFlags().StringVarP(&ackDeadline, ackDeadlineFlagName, "t", ackDeadline, "pubsub ack deadline(unit seconds)")
```

Comment:
Since this is not a global option, could you change to add the flag to the register_push command?

https://github.com/k-yomo/pubsub_cli/blob/master/cmd/register_push.go#L37

Reply:
I think the initial letter `a` for  short flag name would be more intuitive!